src/kontrol/__main__.py

```diff
@@ -62,6 +64,14 @@ def main() -> None:
     args = parser.parse_args()
     logging.basicConfig(level=_loglevel(args), format=_LOG_FORMAT)
 
+    expected_k_version = KVersion.parse(f'v{pyk.K_VERSION}')
+    actual_k_version = k_version()
+
+    if expected_k_version != actual_k_version:
+        print(
+            f'WARNING: An outdated version of K (v{actual_k_version}) is being used. Updating K to v{expected_k_version} is RECOMMENDED'
+        )
```

Comment:
```suggestion
        _LOGGER.warning(
            f'An outdated version of K (v{actual_k_version}) is being used. Updating K to v{expected_k_version} is RECOMMENDED'
        )
```

Reply:
The wording suggests the issue is with an outdated K version, but actually the warning is emitted on a version that does not match the expected one.

Reply:
I addressed this. Please see if it's better now.

Reply:
Looks good.